galley/cmd/galley/cmd/root.go

```diff
@@ -123,6 +124,8 @@ func GetRootCmd(args []string, printf, fatalf shared.FormatFn) *cobra.Command {
 		"Interval of updating file for the Galley readiness probe.")
 	rootCmd.PersistentFlags().UintVar(&monitoringPort, "monitoringPort", 9093,
 		"Port to use for exposing self-monitoring information")
+	rootCmd.PersistentFlags().BoolVar(&enableProfiling, "enableProfiling", true,
```

Comment:
Maybe `false` for default?

Reply:
`pilot`, `mixer` are `true` by default, while `citadel` is `false`.  Do we need to make them to `false` for default?

Reply:
Not sure. Let's wait for community feedback on this because it doesn't seem right for me that profiling is on by default.

Reply:
Thanks for your response. Let us wait for the feedback from community.

Reply:
I don't know. It is on in the initial version. @kyessenov any thoughts?

Reply:
Let's keep it off by default. We originally made this default=true in Pilot ~2 years ago to help with early debugging before Istio was used in production.

Reply:
make it to default=false for `mixer`, `pilot`, `galley` and `citadel`. right?